src/__internal__/validation-message/validation-message.component.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styledSystemPropTypes from "@styled-system/prop-types";
+import PropTypes from "prop-types";
+import { filterStyledSystemMarginProps } from "../../style/utils";
+import StyledValidationMessage from "./validation-message.style";
+
+const ValidationMessage = ({ error, warning, info }) => {
+  const validation = error || warning || info;
+  const isStringValidation = typeof validation === "string";
+
+  return (
```

Comment:
We shouldn't be returning an empty fragment if `!isStringValidation`. I would turn the logic around to return null in that case: 

```jsx
const ValidationMessage = ({ error, warning, info }) => {
  const validation = error || warning || info;
  const isStringValidation = typeof validation === "string";

  if (!isStringValidation) return null;

  return (
    <>
      {error && <StyledValidationMessage>{error}</StyledValidationMessage>}
      {warning && !error && (
        <StyledValidationMessage type="warning">
          {warning}
        </StyledValidationMessage>
      )}
      {info && !warning && !error && (
        <StyledValidationMessage type="info">{info}</StyledValidationMessage>
      )}
    </>
  );
};
```